### PR TITLE
docs: create a release guide

### PR DESCRIPTION
**Pull request by MartinWeb**
Create a release guide.

### CODE REVIEW

**Review by pplancq — DISMISSED** [review]

**Review by fffan64 — DISMISSED** [review]

**Review by a-pourrier — DISMISSED** [review]

**Review by samuel-gomez-axa — DISMISSED** [review]

File: RELEASE-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# How to release
+
+This documentation explains the process to release the design system.
+
+## Create a release candidate
+
+First of all, before releasing a new version, you need to create a release candidate. This is a version that is not yet released but that can be tested by the users.
+
```
Comment (JLou) [inline RELEASE-GUIDE.md]:
```suggestion
First of all, before releasing a new version, you need to create a release candidate. A RC is not stable yet but can be tested by the users.

```

File: RELEASE-GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+To create a release candidate, you need to create a new branch from the `main` branch. The branch name should be `releases-DESIGN_SYSTEM/X.Y.Z` where `DESIGN_SYSTEM` is the design system you want to release and `X.Y.Z` is the version number you want.
+
```
Comment (JLou) [inline RELEASE-GUIDE.md:10]: Donnner le nom exact des branches (confusion possible entre agent/slash ou look-and-feel/lookandfeel/apollo)

File: RELEASE-GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+For example, if you want to make a release for `slash` design system, you should create a branch named `releases-slash/1.0.0`.
+
+```bash
+git switch main
+git pull
+git switch -c releases-slash/1.0.0-RC.1
+```
+
+If it's the first time you release candidate this new version, you should'nt need to commit anything. But you can still commit some changes on this branch, for example, after testing the first release candidate and realizing that you need to fix something.
+When you are ready to candidate release the version, you can create a tag.
```
Comment (JLou) [inline RELEASE-GUIDE.md]:
```suggestion
If it's the first time you release candidate this new version, you don't need to commit anything. But you can still commit some changes on this branch, for example, after testing the first release candidate and realizing that you need to fix something.
When you are ready to candidate release the version, you can create a tag.
```

**Review by JLou — CHANGES_REQUESTED** [review]

File: RELEASE-GUIDE.md (hunk 2 above)
Comment (MartinWeb) [reply RELEASE-GUIDE.md:10]: done

**Review by JovaanB — APPROVED** [review]

**Review by samuel-gomez-axa — APPROVED** [review]

**Review by JLou — APPROVED** [review]

**Pull request merged**